leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/HeapAnalysisListScreen.kt

```diff
@@ -66,21 +68,11 @@ internal class HeapAnalysisListScreen : Screen() {
               DateUtils.FORMAT_SHOW_TIME or DateUtils.FORMAT_SHOW_DATE
           )
 
-          val title =
-            when {
-              projection.exceptionSummary != null -> projection.exceptionSummary
-              else -> {
-                val quantityFormat = resources.getQuantityString(
-                    R.plurals.leak_canary_heap_analysis_list_retained_instances,
-                    projection.retainedInstanceCount
-                )
-                String.format(
-                    quantityFormat,
-                    projection.retainedInstanceCount
-                )
-              }
-            }
-          titleView.text = String.format("$index. %s", title)
+          val title = projection.exceptionSummary ?: resources.getQuantityString(
```

Comment:
`getQuantityString()` has an overload that takes the String.format arguments as last param.